### ISSUE TITLE
PhoneInput does not render flag emojis in Windows

### ISSUE THREAD

**Issue by hhvys** [uber/baseweb#4675]
Windows
![Windows_PhoneInput](https://user-images.githubusercontent.com/33412821/146880634-cd6e349b-91f2-4d3e-a543-f37f42e062ef.png)

Mac
<img width="1792" alt="Mac_PhoneInput" src="https://user-images.githubusercontent.com/33412821/146880683-e7b48c4b-6265-45fb-a91a-b23b706249fa.png">

In windows, country code is visible instead of flags.

## Expected Behavior

The UX should be the same irrespective of the environment.

## Your Environment

| Tech    | Version |
| ------- | ------- |

| Base UI | v10.6.0  |
| React   | 17.0.2 |
| browser | Chrome (Windows) |

- [ ] I have searched the [issues](https://github.com/uber/baseweb/issues) of this repository and believe that this is not a duplicate.

**Comment by bsplosion** [comment]
Any update on this?  Having environment pivots like this is unexpected behavior.

**Comment by jacquesg** [comment]
Experiencing the same issue as well.

**Comment by jacquesg** [comment]
I've managed to get this working by following this guide: https://github.com/talkjs/country-flag-emoji-polyfill

TLDR: Chrome on Windows simply doesn't do flag emojis and there are no plans to support it by either Google/Microsoft.
